plugins/inputs/webhooks/papertrail/papertrail_webhooks.go

```diff
@@ -64,7 +66,7 @@ func (pt *PapertrailWebhook) eventHandler(w http.ResponseWriter, r *http.Request
 			}
 			pt.acc.AddFields("papertrail", fields, tags, e.ReceivedAt)
 		}
-	} else if payload.Counts != nil {
+	} else if payload.Counts != nil { //nolint: revive // "Simplifying" if c {...} else {... return } would not simplify anything at all in this case
```

Comment:
You can simplify be switching the conditional `else if payload.Counts == nil {`, but I'm not sure we should do this. If not we maybe should say so in the comment e.g. `// We are not simplifying here to keep the structure...` or similar.

Reply:
Changed.